app/lib/steps/logic.rb

```diff
@@ -88,6 +88,10 @@ def owns_property_shared_ownership?(session_data)
         !skip_capital_questions?(session_data) && session_data["property_owned"] == "shared_ownership"
       end
 
+      def landlord_is_not_the_only_joint_owner?(session_data)
```

Comment:
this scans weirdly for me, can we rename this method so we dont have the NOT, so for example, `property_has_more_than_two_owners?` or `multiple_party_ownership?`